pkgcfg/package-configuration/ruby*.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "ruby",
+  "version": "0.0.1",
+  "env": {
+    "RUBY_CONFDIR": "{{ .DevboxRoot }}/conf/ruby",
+    "GEMRC": "{{ .DevboxRoot }}/conf/ruby/.gemrc",
+    "GEM_HOME": "{{ .DevboxRoot }}/conf/ruby"
+  }
```

Comment:
@ipince you've had recent ruby experience, so could you double check this? Thanks!

Reply:
Hmm... well I didn't really mess with setting these vars, so I'm not sure. Seems good to me though :)